client/src/containers/CategoryType.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const CategoryType = ({match}) => {
+    return (
+        <div>
+            <h1>{match.params.type}</h1>
+        </div>
+    )
+}
```

Comment:
pass the type by Default if we are not dealing with any other properties of the match variable.

Reply:
I am going to work on category in next PR